### PR TITLE
feat(): remove depricated bal-timeinput component, closes #202

### LINKED ISSUE #202: Rebrand Bal-Time
Rebrand and refactor the time input to match the new concept with masked inputs.

One team internally is using this component.
@alexsoeres 

- [x] Check with alex if it is still needed and how we can rebrand it without a breaking change.

### PR DESCRIPTION
**Pull request by nobilo**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: packages/components/.build/stencil/stencil.bindings.angular.ts [hunk 1]
```diff
@@ -78,13 +78,6 @@ export const AngularGenerator = () =>
           'bal-hint-text',
         ],
       },
-      'bal-stage': {
```
Comment (hirsch88) [inline packages/components/.build/stencil/stencil.bindings.angular.ts:81]: why did you remove the stage ?

**Review by hirsch88 — CHANGES_REQUESTED** [review]

File: packages/components/.build/stencil/stencil.bindings.angular.ts (hunk 1 above)
Comment (nobilo) [reply packages/components/.build/stencil/stencil.bindings.angular.ts:81]: its duplicate, IDE reported :), look at line 66 :)

Comment (hirsch88) [reply packages/components/.build/stencil/stencil.bindings.angular.ts:81]: okay nice thanks

**Review by hirsch88 — APPROVED** [review]
